Makie/src/basic_recipes/waterfall.jl

```diff
@@ -44,16 +44,16 @@ function Makie.plot!(p::Waterfall)
             xy[inds] .= Point2f.(x[inds], fromto.to)
             final[inds] .= Point2f.(x[inds], fromto.final)
         end
-        return (xy = xy, fillto = fillto, final = final)
+        return xy, fillto, final
     end
 
-    fromto = lift(stack_bars, p, p[1], p.dodge, p.stack)
+    map!(stack_bars, p, [:converted_1, :dodge, :stack], [:xy, :fillto, :final])
 
-    if p.show_final[]
-        final_gap = p.final_gap[] === automatic ? p.dodge[] == automatic ? 0 : p.gap : p.final_gap
+    if p.show_final
```

Comment:
Figured out that this is where the error is. Earlier this was `p.show_final[]`, and that converted it to an Observable implicitly, I think.  Is there a way to check the value of the variable without doing that?